phpstan.neon

```diff
@@ -1,6 +1,7 @@
 parameters:
-    checkMissingIterableValueType: false
-    checkGenericClassInNonGenericObjectType: false
+    ignoreErrors:
+        - identifier: missingType.iterableValue
```

Comment:
Can we add comments why the suppresions are needed?

Reply:
yeah, that's a good point, I think the reason might just be due to the fact that this rule was introduced after the code was written so there are a lot of instances of "no value specified on iterable type array". The suppression was already there when we added the linting (and the same in all the other tickets). I'll look into whether we can fix the issues

Reply:
We've decided that to fix the issues highlighted by this suppression is a wide-reaching change - adding typing would require a new ticket to investigate in more detail. We will leave as-is for now